.changeset/hip-cherries-behave.md

```diff
@@ -0,0 +1,7 @@
+---
+"astro": patch
+---
+
+Improves Node.js streaming performance
+
+This uses an AsyncIterable instead of a ReadableStream to do streaming in Node.js. This is a non-standard enhancement by Node, so this is done only in that environment.
```

Comment:
```suggestion
Improves Node.js streaming performance.

This uses an `AsyncIterable` instead of a `ReadableStream` to do streaming in Node.js. This is a non-standard enhancement by Node, which is done only in that environment.
```